src/components/MovieDetails/MovieDetails.js

```diff
@@ -20,23 +20,41 @@ class MovieDetails extends Component {
             .catch(() => this.setState({ error: 'Having trouble finding this movie right now...please try again.'} ));
     }
 
-    // formatOverviewLength = (overview) => {
-    //     if (overview.length && overview.length > 20) {
-    //         const shortOverview = overview.slice(0, 20) + '...';
-    //         console.log('Shorty', shortOverview)
-    //         return shortOverview;
-    //     } else {
-    //         console.log('OVERVIEW', overview)
-    //         return overview;
-    //     }
-    // }
+    formatOverview = (overview) => {
```

Comment:
Good to see you got this working again, so text isn't spilling out of our moviePoster container at different screen sizes. We should still look into the package you found that allows us to add in a linkable piece that would allow the customer to view the rest of the descriptoin.